data/json/recipes/electronics/tools.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "result": "makeshift_oscilloscope",
+    "type" : "recipe",
+    "category": "CSC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_TOOLS",
+    "skill_used": "electronics",
+    "difficulty": 4,
+    "skills_required": [ "fabrication", 2 ],
+    "time": 240000,
+    "book_learn": [ [ "advanced_electronics", 4 ], [ "mag_electronics", 6 ] , [ "manual_electronics", 5 ] ],
```

Comment:
It's not used in any at present. The `multimeter` is the common tool providing `MULTIMETER 2`. The oscilloscopes are required for a higher level of circuit analysis of `MULTIMETER 3`